app/src/main/java/com/example/drawingactivity/CustomColorViewModel.kt

```diff
@@ -29,10 +31,20 @@ class CustomColorViewModel(
         updateViews()
     }
 
+    private var viewModelDrawnBitmap: Bitmap? = null
+
+    fun saveUserBitmap(b: Bitmap?) {
+        viewModelDrawnBitmap = b
+    }
+
+    fun fetchUpdate() {
```

Comment:
KDoc public functions

Reply:
Removed this in favor of making the update function public and calling that to update with the views when needed.
This was just an intermediary that didn't add any value.